core/protos/coreevents.go

```diff
@@ -75,24 +76,13 @@ func NewEventTaskState(taskid,state string) *Event {
 	return WrapEvent(&stCh)
 }
 
-func NewEventTaskStatus(status *mesos.TaskStatus) *Event {
-
-	taskId := status.GetTaskID().Value
-	var statuschanged Ev_TaskStatusChanged
+func NewEventTaskStatus(taskid,status string) *Event {
```

Comment:
I still push the `ACTIVE`/`INACTIVE` and plus the `KILLED` status, I just moved it inside `taskman` and I push it from `updateTaskStatus` instead of the `scheduler.go`.